src/_gettsim/transfers/unterhaltsvors.py

```diff
@@ -317,15 +314,14 @@ def _unterhaltsvors_anspruch_kind_m_ab_201707(
     -------
 
     """
-    altersgrenzen = unterhaltsvors_params["altersgrenzen_bezug"]
     mindestunterhalt = unterhalt_params["mindestunterhalt"]
 
-    if alter < altersgrenzen[1]:
-        out = mindestunterhalt[altersgrenzen[1]] - _kindergeld_erstes_kind_m
-    elif altersgrenzen[1] <= alter < altersgrenzen[2]:
-        out = mindestunterhalt[altersgrenzen[2]] - _kindergeld_erstes_kind_m
-    elif altersgrenzen[2] <= alter < altersgrenzen[3]:
-        out = mindestunterhalt[altersgrenzen[3]] - _kindergeld_erstes_kind_m
+    if mindestunterhalt[1]["min_alter"] <= alter <= mindestunterhalt[1]["max_alter"]:
```

Comment:
I switched to the age thresholds of the Mindestunterhalt here, because the value of the transfer is taken from there as well. Makes it less error prone in the future imo.

Then, the age thresholds in the UV params file are not used after 2016, so I removed the 2017 reform from there.

Do you agree?